Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml

```diff
@@ -1867,6 +1868,405 @@ tasks:
           "y": 1470
         }
       }
+  "64":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                filters:
+                - - left:
+                      iscontext: true
+                      value:
+                        simple: AWS.SSM.InventoryEntry.Entries.[0].InstanceStatus
+                    operator: isEqualString
+                    right:
+                      value:
+                        simple: Active
+                root: AWS.SSM
+          operator: isNotEmpty
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "64"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "72"
+      "yes":
+      - "65"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if there is SSM information to set in the system IDs field.
+      id: 31a242a0-5a16-433c-8034-e9712abe8f9a
+      iscommand: false
+      name: Is there AWS SSM information?
+      type: condition
+      version: -1
+    taskid: 31a242a0-5a16-433c-8034-e9712abe8f9a
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 40,
+          "y": 515
+        }
+      }
+  "65":
+    continueonerrortype: ""
+    id: "65"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "66"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmsystemids
+      keys:
+        simple: type,id,link
+      val1:
+        simple: ASSET-TYPE
+      val2:
+        simple: AWS-SSM
+      val3:
+        simple: n/a
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: Sets the type of cloud asset to the grid field for the ASM system IDs object.
+      id: 627c9db8-e0bf-4d1e-87c2-de7a9b566d8a
+      iscommand: false
+      name: Set system IDs grid field (type)
+      script: GridFieldSetup
+      type: regular
+      version: -1
+    taskid: 627c9db8-e0bf-4d1e-87c2-de7a9b566d8a
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 40,
+          "y": 720
+        }
+      }
+  "66":
+    continueonerrortype: ""
+    id: "66"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "67"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmsystemids
+      keys:
+        simple: type,id,link
+      val1:
+        simple: ASSET-SSM-AGENT-STATUS
+      val2:
+        simple: ${AWS.SSM.InventoryEntry.Entries.[0].InstanceStatus}
```

Comment:
may I suggest using a transformer of the first array element instead of pointing to a position in the list?
If this structure is changed in the future or the returned element will be only 1 value the context path will be different.

Reply:
Thank you for the suggestion. Updated the task to use FirstArrayElement.